CHANGELOG.md

```diff
@@ -10,6 +10,11 @@ bumped for multiple releases during one month.
 <!-- BEGIN RELEASE NOTES -->
 ### [Unreleased]
 
+### [24.1.0] - 2024-01-18
+
+#### Added
+- Added X-Klaviyo-User-Agent header to KlaviyoEventService requests
```

Comment:
tiniest of nits: you added this to the event service and the profile service. Can probably just say "Klaviyo API requests"

Reply:
Good catch. Thank you 🥇